plugins/nvm/nvm.plugin.zsh

```diff
@@ -1,3 +1,4 @@
 # The addition 'nvm install' attempts in ~/.profile
 
 [[ -s ~/.nvm/nvm.sh ]] && . ~/.nvm/nvm.sh
+[[ -r $NVM_DIR/bash_completion ]] && . $NVM_DIR/bash_completion
```

Comment:
You can check if bashcompinit is defined with `which bashcompinit`. 
You can also delete the zcomdump files (`rm ~/.zcomdump*`) and restart the terminal to see if it works. 
It can also be that nvm.sh is doing that for us, you could have a look there.

Finally, it is surprising to me how only the `compleat` plugin has a bashcompinit statement, nowhere else it is found in the oh-my-zsh repository! 
Let's see what you can find!